### DISCUSSION TITLE
PathSegments Add Ctor overload

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#16786] [Q&A] [answered]
Hi,
Can I create a PR that adds Ctor with capacity argument to `PathSegments`?

**Comment by kekekeks** [comment, marked as answer]
[Yes](https://learn.microsoft.com/en-us/dotnet/api/system.windows.media.pathsegmentcollection.-ctor?view=windowsdesktop-8.0#system-windows-media-pathsegmentcollection-ctor(system-int32))
